### PR TITLE
=pro add log timestamps to all sbt output and enable sbt debug logging for multi-jvm for ssh debugging

### PR DESCRIPTION
**Pull request by jrudolph**
Increases logging output but might help seeing exactly where time is lost

screenshot from 2018-02-05 17-55-24

### CODE REVIEW

File: project/AddLogTimestamps.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  override def requires: Plugins = plugins.JvmPlugin
+  override def trigger: PluginTrigger = allRequirements
+
+  override def projectSettings: Seq[Def.Setting[_]] = {
+    logManager := {
+      val original = logManager.value
+
+      if (enableTimestamps) {
+        val myOut = new PrintWriter(System.out) {
+          val dateTimeFormat = new SimpleDateFormat("MM-dd HH:mm:ss.SSS")
```
Comment (patriknw) [inline project/AddLogTimestamps.scala]: could this be called concurrently? perhaps use `DateTimeFormatter`, since `SimpleDateFormat` is not thread safe

**Comment by patriknw** [comment]
wouldn't hurt to include this, even though the immediate problem is solved, or what do you think @jrudolph ?

**Comment by jrudolph** [comment]
I changed it to use `DateTimeFormatter`. I wonder though if it is not too verbose for common usage? It will prefix every log line from sbt with the current time. Might be worth a try though...

**Comment by jrudolph** [comment]
PLS BUILD

**Comment by jrudolph** [comment]
I added the flag on the jenkins PR validator so we can determine whether the extra output is worth it.

**Comment by jrudolph** [comment]
You can now have a look at https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/11969/consoleFull to see whether you think if it's overkill or not.

**Review by patriknw — APPROVED** [review]
LGTM
Looked at the jenkins log, I think it can be useful, so let's try.

**Pull request merged**